addons/docs/src/frameworks/common/preset.ts

```diff
@@ -34,8 +31,22 @@ function createBabelOptions({ babelOptions, mdxBabelOptions, configureJSX }: Bab
   };
 }
 
-export function webpack(webpackConfig: any = {}, options: any = {}) {
+export async function webpack(
+  webpackConfig: any = {},
+  options: Options &
+    BabelParams & { sourceLoaderOptions: any; transcludeMarkdown: boolean } & Parameters<
+      typeof createCompiler
+    >[0]
+) {
+  // @ts-ignore
+  const { builder = 'webpack4' } = await options.presets.apply('core', {});
+
+  const resolvedBabelLoader = require.resolve('babel-loader', {
+    paths: [require.resolve(`@storybook/builder-${builder}`)], // FIXME!!!
```

Comment:
What if the `builder` is `storybook-builder-vite`? Also what about the `FIXME`?

Reply:
fixme can be removed, it's fixed.

I didn't think of the `storybook-builder-vite`.